test_vm/tests/verified_claim_test.rs

```diff
@@ -428,3 +428,133 @@ fn expired_allocations_test<BS: Blockstore>(v: &dyn VM<BS>) {
 
     expect_invariants(v, &[invariant_failure_patterns::REWARD_STATE_EPOCH_MISMATCH.to_owned()]);
 }
+
+#[test]
+fn deal_passes_claim_fails() {
```

Comment:
As part of efforts to make these integration tests runnable on a real VM (e.g. for gas profiling) there's a new entry point convention here. Please follow the pattern used in the other tests in this file. If for some reason the `VM` trait doesn't have functionality you need for this test, please comment why and reference the issue https://github.com/filecoin-project/builtin-actors/issues/1294.